packages/airnode-admin/test/setup/init/set-define-property.ts

```diff
@@ -2,15 +2,17 @@
 // and use this function as a workaround at the top of your test.
 //
 // Credit: https://github.com/facebook/jest/issues/6914#issuecomment-654710111
-const { defineProperty } = Object;
-Object.defineProperty = function (object, name, meta) {
-  if (meta.get && !meta.configurable) {
-    // it might be an ES6 exports object
-    return defineProperty(object, name, {
-      ...meta,
-      configurable: true, // prevent freezing
-    });
-  }
+{
```

Comment:
In my IDE, if I ask it to "recompile" it indicates that `defineProperty` is redeclared. Adding some braces fixes it, but my IDE is possibly missing some configuration.

Reply:
I don't think this should be included in this PR - it is likely an issue of your IDE.
If this is something that is broken - create a new PR for it